src/events/twitch/twitchStreamOffline.ts

```diff
@@ -22,8 +21,12 @@ export default class extends Event {
 			if (typeof guild === 'undefined') continue;
 
 			// Synchronize the settings, then retrieve to all of its subscriptions
-			await guild.settings.sync();
-			const subscriptions = guild.settings.get(GuildSettings.Notifications.Streams.Twitch.Streamers).find(([id]) => id === streamer.id);
+			const { allSubscriptions, language } = await guild.readSettings((settings) => ({
+				allSubscriptions: settings.notificationsStreamsTwitchStreamers,
+				language: settings.getLanguage()
+			}));
```

Comment:
How come this doesn't use the `[SettingsKeys]` syntax?
```suggestion
			const [allSubscriptions, language] = await guild.readSettings((settings) => [
				settings.notificationsStreamsTwitchStreamers,
				settings.getLanguage()
			]);
```